firstvoices/backend/permissions/filters/view.py

```diff
@@ -5,6 +5,8 @@
 #
 
 
+# Note that this view filter is re-created in the following file to work with ElasticSearch:
+# firstvoices/apps/fv_search/permissions.py
```

Comment:
Love this but I think it can be even more intense. Can you please add some kind of direct instruction, like "If you change this, you need to make matching changes to the search permissions and to the predicate"